launch/ros_control_test.launch

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0"?>
 <!DOCTYPE launch SYSTEM "https://gist.githubusercontent.com/nalt/dfa2abc9d2e3ae4feb82ca5608090387/raw/roslaunch.dtd">
 <launch>
-    <node pkg="wrevolution" type="RosControlTest" name="RosControlTests" output="screen" />
+    <!-- TODO (Ben Nowotny) we should not condition on this variable in here, should be in a
+    separate package-->
+    <node pkg="wrevolution" type="RosControlTest" name="RosControlTests" output="screen"
+        if="$(eval env('WROVER_HW') == 'REAL')" />
+    <node pkg="wrevolution" type="MockMotors" name="RosControlTests-Mock" output="screen"
+        unless="$(eval env('WROVER_HW') == 'REAL')" />
```

Comment:
Needs a TODO to resolve, this package should parameterize this rather than depend on the environment variable directly.

Reply:
This TODO already exists.